web-client/src/components/information.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import '../App.css';
+
+function Information(props) {
+    const styles = makeStyles({
+        main:
+        {
+            padding: '5%',
+            width: '60%',
+        },
+        left:
+        {
+            width: '30%',
+            float: 'left',
+        },
+        right:
+        {
+            float: 'left',
+            marginTop: '2%',
+        },
+        photo:
+        {
+            display: 'block',
+            width: '60%',
+            borderRadius: '50%',
+        },
+        text:
+        {
+            fontWeight: 'normal',
+        },
+        a:
+        {
+            textDecoration: 'none',
+            color: 'black',
+
+            '&:hover' :
+            {
+                color: 'red',
+            }
```

Comment:
Please don't use red color if it's not an error :c
You can use light grey/grey instead

Reply:
ok I fixed my mistakes